integration/tests/epochs/recover_epoch/recover_epoch_efm_test.go

```diff
@@ -19,16 +23,79 @@ type RecoverEpochSuite struct {
 	Suite
 }
 
-// TestRecoverEpoch ensures that the recover_epoch transaction flow works as expected. This test will simulate the network going
-// into EFM by taking a consensus node offline before completing the DKG. While in EFM mode the test will execute the efm-recover-tx-args
-// CLI command to generate transaction arguments to submit a recover_epoch transaction, after submitting the transaction the test will
-// ensure the network is healthy.
+// TestRecoverEpoch ensures that the recover epoch governance transaction flow works as expected and a network that
```

Comment:
Additionally we need to have a test which triggers EFM in committed phase and verifies:
1. if we correctly enter committed epoch
2. next epoch should progress in EFM mode and add epoch extensions
3. after processing epoch recover we should be able to transition to the next(recovered) epoch.

Reply:
To trigger EFM in committed phase we can wait for the epoch to become committed and then submit a recovery transaction which is not expected on happy path.

Reply:
Yes that will be implemented in part 2 https://github.com/onflow/flow-go/issues/6164 there are still some downstream components that need to be updated.